### ISSUE TITLE
ePSXe

### ISSUE THREAD

**Issue by Mickevincent** [libretro/common-shaders#76]
Just one question Hyllian, is this possible to get for ePSXe?

**Comment by Hyllian** [comment]
What are you referring to?

**Comment by Hyllian** [comment]
Ok.

No it isn't possible, because ePSXe doesn't support multipass shaders.

Anyway, in the near future the Mednafen-Bettle core will have
hard-acceleration, which will put it probably ahead of ePSXe in IQ and
compatibility.

**Comment by Mickevincent** [comment]
Ah, alright, thanx!

**Comment by twinaphex** [comment]
Hyllian - actually, the GL renderer is in the Mednafen PSX core right now.

**Comment by Hyllian** [comment]
Ok. But, is it still buggy? (I didn't test it lately.)

**Comment by twinaphex** [comment]
Yes, it's not finished but it's a lot better than what it was before.

And it can also still be slow in some places but optimization is supposed to be done later.

It might already be good enough for you to try to base some tests around it, you could maybe advise simias on things like texture filtering with xBRz or something similar.
